README.md

```diff
@@ -195,6 +199,19 @@ After having imported cloudbreak repo root you can launch Datalake application b
 -Ddatalake.cloudbreak.url=http://localhost:8080
 ````
 
+### Running environment in IDEA
+
+After having imported cloudbreak repo root you can launch Environment application by executing the com.sequenceiq.environment.EnvironmentApplication class with the following VM options:
+
+````
+-Denvironment.cloudbreak.url=http://localhost:8080
```

Comment:
a lot of these are already in the application.yml so these vm options are not needed.

https://github.com/hortonworks/cloudbreak/blob/668be14519ad3cd5c152413c7fb26b0b501ac269/environment/src/main/resources/application.yml#L30-L47